server/memstore_test.go

```diff
@@ -90,6 +91,67 @@ func TestMemStoreMsgLimit(t *testing.T) {
 	}
 }
 
+func TestMemStorePerformance(t *testing.T) {
```

Comment:
Let's move this to norace_test.go and follow correct naming conventions there.

Reply:
done